.gen/pipeline/api/openapi.yaml

```diff
@@ -18500,13 +18500,57 @@ components:
       - name
       - roles
       type: object
+    CreatePKEOnVsphereClusterRequest:
+      allOf:
+      - $ref: '#/components/schemas/CreatePKEClusterRequestBase'
+      - $ref: '#/components/schemas/CreatePKEOnVsphereClusterRequest_allOf'
+    PKEOnVsphereNodePool:
+      properties:
+        name:
+          type: string
+        roles:
+          items:
+            enum:
+            - master
+            - system
+            - worker
+            type: string
+          type: array
+        labels:
+          additionalProperties:
+            type: string
+          type: object
+        count:
+          minimum: 1
+          type: integer
+        vcpu:
+          description: Number of VCPUs to attach to each node.
+          example: 2
+          type: integer
+        ramMB:
```

Comment:
Can this be just `ram`?

Reply:
it tends to cause confusion if you expect a field's value in an unusual unit without mentioning it in the name. the only reason for expecting this value in MBs is because vsphere handles it on this granularity.

Reply:
There are various solutions to mitigate the unusual unit problem:

- documentation
- examples
- expect a string representation (like in case of Kubernetes)
- use an "expected" unit and convert it